testing/zerotier-one/zerotier-one.pre-install

```diff
@@ -0,0 +1,4 @@
+#!/bin/sh
+
+/usr/sbin/addgroup -S zerotier-one
+/usr/sbin/adduser -D -H -S -h /var/lib/zerotier-one -s /sbin/nologin -g 'https://www.zerotier.com/' -G zerotier-one zerotier-one
```

Comment:
The daemon seems to drop privileges to this hardcoded user: https://github.com/zerotier/ZeroTierOne/blob/fe5257df81c4ec4b5d48f707eb794de0748b7ac0/one.cpp#L958